src/pages/public/imgae-gallery/Gallery.js

```diff
@@ -1,22 +1,26 @@
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 import { Grid } from "@mui/material";
 import AppLoader from "@components/Loader/AppLoader";
+import { API, NetworkManager } from "@network/core";
 function Gallery() {
-  const [Gallery, setGallery] = useState(null);
-  useEffect(() => {
-    const urlNew = "https://picsum.photos/v2/list";
-    fetch(urlNew)
-      .then((resp) => resp.json())
-      .then((resp) => setGallery(resp));
-  }, []);
+  const [gallery, setGallery] = useState(null);
+
+  const instanceOfNetwork = new NetworkManager(API.GALLERY.DATA);
+  instanceOfNetwork
+    .httpRequest(false)
+    .then((res) => res.data)
+    .then((data) => {
```

Comment:
what's the point of nesting up the promises here? You should have got the correct JSON on the first .then param.